doc/en/announce/sprint2016.rst

```diff
@@ -39,8 +39,6 @@ Over 20 participants took part from 4 continents, including employees
 from Splunk, Personalkollen, Cobe.io, FanDuel and Dolby. Some newcomers
 mixed with developers who have worked on pytest since its beginning, and
 of course everyone in between.
-    Ana Ribeiro, Brazil
```

Comment:
I think it was decided to remove the entire participants list at some point, but those two names seem to be have left by accident (they were a syntax error in Sphinx)